com.unity.perception/Documentation~/GroundTruth/KeypointLabeler.md

```diff
@@ -58,6 +59,16 @@ The Keypoint Labeler captures keypoints each frame from each object in the scene
 
 For a tutorial on setting up your project for keypoint labeling, see the [Human Pose Labeling and Randomization Tutorial](../HPTutorial/TUTORIAL.md).
 
+## Keypoint Self Occlusion
+A keypoint will be considered not visible if it outside of the camera's view, the camera cannot see it because
+another object is occluding it (in between the camera and the keypoint), or another part of its own model is occluding it (for
+example a human model's arm is raised and blocking its face from view). The keypoint is a point in space generally internal
+to the model, for example the elbow joint is in the center of the arm volume. A self occlusion distance value has
+been added to each keypoint allowing the keypoint to have depth. If a keypoint is occluded by itself, then this value is added to keypoint location to see
```

Comment:
Use present tense (ex. 'exists', 'has') instead of past tense ('has been added')